src/api-requests/get-articles.ts

```diff
@@ -114,3 +116,21 @@ export const getAllArticlesFromAPI = async (
 
     return data.articles;
 };
+
+export const getAllDraftArticlesFromAPI = async (
+    client: UnderlyingClient<'ApolloREST'>,
+    calculatedSlug: string
+): Promise<Article[]> => {
+    calculatedSlug = '"' + calculatedSlug + '"';
+    const query = gql`
+        query Articles {
+            articles(_publicationState : "preview", calculated_slug : ${calculatedSlug})
+            @rest(type: "Article", path: "/new-articles?{args}") {
+                ${fields}
+            }
+        }`;
+    const { data }: ApolloQueryResult<{ articles: Article[] }> =
+        await client.query({ query, fetchPolicy: 'no-cache' });
```

Comment:
fetchPolicy: 'no-cache' - will always make network call for this query. it is needed since the users want to see the changes in live and there is no room for stale data.